api/openapi/openapi_metadata.py

```diff
@@ -0,0 +1,31 @@
+openapi_metadata = {
+    "title": "Observations Europe EUMETNET",
+    "description": (
+        "EDR data service for European environmental observations from EUMETNET," " co-funded by the European Union"
+    ),
+    "contacts": [
```

Comment:
According to [this](https://fastapi.tiangolo.com/reference/fastapi/#fastapi.FastAPI), there is no `contacts` parameter, only `contact`.

It also doesn't show up anywhere in the output, as far as I can see.